autogen/safer-cluster/main.tf.tmpl

```diff
@@ -107,7 +107,7 @@ module "gke" {
   cluster_resource_labels = var.cluster_resource_labels
 
   // We enable private endpoints to limit exposure.
-  enable_private_endpoint       = true
```

Comment:
This has the opposite effect of what its name implies. In order to follow the GKE hardening guides recommendation at https://cloud.google.com/kubernetes-engine/docs/how-to/hardening-your-cluster#restrict_network_access_to_the_control_plane_and_nodes ("Public endpoint access enabled, master authorized networks enabled"), this `enable_private_endpoint` needs to be set to `false`.


See also https://stackoverflow.com/a/58651940/6720110